sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -35,14 +35,21 @@ const defaultConsumerClientOptions: Required<Pick<
 };
 
 /**
- * The `EventHubConsumerClient` is the main point of interaction for consuming events in Azure Event Hubs service.
+ * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+ * Event Hub instance.
  *
  * There are multiple ways to create an `EventHubConsumerClient`
  * - Use the connection string from the SAS policy created for your Event Hub instance.
  * - Use the connection string from the SAS policy created for your Event Hub namespace,
  * and the name of the Event Hub instance
- * - Use the fully qualified domain name of your Event Hub namespace like `<yournamespace>.servicebus.windows.net`,
+ * - Use the fully qualified version of the namespace like `<yournamespace>.servicebus.windows.net`,
  * and a credentials object.
+ * 
+ * Optionally, you can also pass the below:
+ * - An options bag to configure the retry policy or proxy settings.
+ * - A checkpoint store used to store checkpoints. These are the positions from where your application
```

Comment:
Wordsmithing attempt:
```suggestion
 * - A checkpoint store used to store checkpoints. Checkpoints contain the position from where your application
```

Reply:
Updated to

> A checkpoint store that is used by the client to read checkpoints to determine the position from where it should resume receiving events when your application gets restarted. It is also used by the client to load balance multiple instances of your application.